### PR TITLE
migrate destinations to use airbyte protocol

### PR DESCRIPTION
**Pull request by cgardens**
This is mostly just find/replace type of operations.

### CODE REVIEW

**Review by sherifnada — APPROVED** [review]
LGTM once build pas

File: airbyte-commons/src/main/java/io/airbyte/commons/time/Instants.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.commons.time;
+
+public class Instants {
+
+  public static long millisToSeconds(long millis) {
```
Comment (michel-tricot) [inline airbyte-commons/src/main/java/io/airbyte/commons/time/Instants.java:29]: You can call it `toSeconds` as long as the argument is called millis.

**Review by michel-tricot — APPROVED** [review]

File: airbyte-commons/src/main/java/io/airbyte/commons/time/Instants.java (hunk 1 above)
Comment (cgardens) [reply airbyte-commons/src/main/java/io/airbyte/commons/time/Instants.java:29]: 👍

File: airbyte-commons/src/main/java/io/airbyte/commons/time/Instants.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.commons.time;
+
+public class Instants {
+
+  public static long toSeconds(long millis) {
```
Comment (jrhizor) [inline airbyte-commons/src/main/java/io/airbyte/commons/time/Instants.java:29]: Is this necessary when `long seconds = TimeUnit.MILLISECONDS.toSeconds(millis);` is part of the standard library?

**Review by jrhizor — APPROVED** [review]

File: airbyte-commons/src/main/java/io/airbyte/commons/time/Instants.java (hunk 2 above)
Comment (cgardens) [reply airbyte-commons/src/main/java/io/airbyte/commons/time/Instants.java:29]: fair enough.

**Pull request merged**